defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# defaults file for sanity-check
+
+SANITY_CHECK_DISK_SPACE:
+    # Mount point
+  - mount: /
```

Comment:
In the current implementation no. But this can be revorked easily.

However we don't want to check all mounted filesystems, `df -h` shows the following: 

```
udev            1.9G   12K  1.9G   1% /dev
tmpfs           386M  372K  386M   1% /run
/dev/vda1        20G  3.2G   16G  17% /
none            4.0K     0  4.0K   0% /sys/fs/cgroup
none            5.0M     0  5.0M   0% /run/lock
none            1.9G     0  1.9G   0% /run/shm
none            100M     0  100M   0% /run/user
```

And I'm not sure what to do if `/run/lock` runs out of space. So maybe monitor just drives filesystems with `/dev`?